src/scrapers/factory.js

```diff
@@ -25,8 +25,8 @@ export default function createScraper(options) {
       return new OtsarHahayalScraper(options);
     case 'visaCal':
       return new VisaCalScraper(options);
-    case 'leumiCard':
-      return new LeumiCardScraper(options);
+    case 'max':
```

Comment:
Why not keeping the `leumiCard` case, and printing a deprecation warning?

Reply:
@baruchiro 
you mean only keep the "leumiCard" case for backward compatibility (with deprecation warning)? and create a new case with "max"?

Reply:
Yes, I don't know if the next syntax is correct, but something like this:

```node
case 'leumiCard':
  console.log(...);
case 'max':
  return new LeumiCardScraper(options);
```

Reply:
So the `case 'leumiCard'` will return `LeumiCardScraper`

Reply:
thanks.
since the COMPANY_ID is hard coded in the test files, we will have to find a work around for this.

Reply:
@baruchiro 
made the necessary changes and kept the 'leumiCard' option.
all tests are looking good.
let me know what you think

Reply:
In my understanding, the project reveals two things:
A list of the **Scrapers**, and the **Factory**.

I think if someone wrote the code based on the **Scrapers** list, if you rename one of them, their code wouldn't change, because it would automatically fit.

The problem is if someone uses the **Factory** directly by sending a string with the value `leumiCard`. In this case, I think it's still worth supporting.

Therefore, I suggest that you remove any reference to `LeumiCard`, from both the **Scrapers** list (then you will need to delete the unnecessary file `leumi-card.test.js`) and the **README** file, and the only reference you leave is in the **Factory**.

@eshaham That's my opinion, but I think we might both be right. Waiting for your decision.

Reply:
@eshaham can we get your input here?

Reply:
@adiohana @baruchiro we should remove the test file for sure.
for backward compatibility, I like the approach above, of keeping support for `leumiCard` but printing a warning.
we can do that until we reach a major version.